lib/plugins/inventory.js

```diff
@@ -20,10 +22,11 @@ function inject (bot, { version, hideErrors }) {
     bobberId = mcData.entitiesByName.fishing_bobber.id
   }
 
+  bot._internalEvents.eating = false
```

Comment:
why is this stored in this _internalEvents thing? it seems it wasn't needed before